### PR TITLE
Fix reference to default Linux image (now 14.04).

### PR DESCRIPTION
**Pull request by felicianotech**
This doc referenced Ubuntu 12.04 as the default Linux image, which is no longer the case.

### CODE REVIEW

File: jekyll/_docs/build-image-trusty.md [hunk 1]
```diff
@@ -3,11 +3,14 @@ layout: classic-docs
 title: Ubuntu 14.04 (Trusty)
 categories: [build-images]
 description: Ubuntu 14.04 (Trusty)
+changefreq: "weekly"
```
Comment (smart-alek) [inline jekyll/_docs/build-image-trusty.md]: What is this for?

File: jekyll/_docs/build-image-trusty.md [hunk 2]
```diff
@@ -3,11 +3,14 @@ layout: classic-docs
...
 ---
 
-You can run your Linux builds on Ubuntu 14.04 Trusty (default is Ubuntu 12.04). You can switch to Trusty from "Project Settings" -> "Build Environment" of your project.
+You can run your Linux builds on Ubuntu 14.04 Trusty (the default). You can 
+switch to Trusty from "Project Settings" -> "Build Environment" of your project.
 
-Please note that you need to trigger a build by pushing commits to GitHub or Bitbucket (instead of rebuilding) to apply the new setting.
+Please note that you need to trigger a build by pushing commits to GitHub or 
+Bitbucket (instead of rebuilding) to apply the new setting.
```
Comment (smart-alek) [inline jekyll/_docs/build-image-trusty.md]: Why split across 2 lines?

File: jekyll/_docs/build-image-trusty.md [hunk 3]
```diff
@@ -3,11 +3,14 @@ layout: classic-docs
 title: Ubuntu 14.04 (Trusty)
 categories: [build-images]
 description: Ubuntu 14.04 (Trusty)
+changefreq: "weekly"
 ---
 
-You can run your Linux builds on Ubuntu 14.04 Trusty (default is Ubuntu 12.04). You can switch to Trusty from "Project Settings" -> "Build Environment" of your project.
+You can run your Linux builds on Ubuntu 14.04 Trusty (the default). You can
```
Comment (smart-alek) [inline jekyll/_docs/build-image-trusty.md]: Change to:

You can run your Linux build on the default, Ubuntu 14.04 Trusty. You can switch to Trusty from "Project Settings" -> "Build Environment" of your project.

File: jekyll/_docs/build-image-trusty.md (hunk 1 above)
Comment (felicianotech) [inline jekyll/_docs/build-image-trusty.md]: This is a signal to the generated sitemap (sitemap.xml for Google) on approximately how often the page might be updated. This in turn hints to search engines like Google how often their search spiders should come back and check the page.

File: jekyll/_docs/build-image-trusty.md (hunk 2 above)
Comment (felicianotech) [inline jekyll/_docs/build-image-trusty.md]: https://news.ycombinator.com/item?id=5436993

File: jekyll/_docs/build-image-trusty.md [hunk 4]
```diff
@@ -6,8 +6,9 @@ description: Ubuntu 14.04 (Trusty)
 changefreq: "weekly"
 ---
 
-You can run your Linux builds on Ubuntu 14.04 Trusty (the default). You can 
-switch to Trusty from "Project Settings" -> "Build Environment" of your project.
+You can run your Linux builds on the default image, Ubuntu 14.04 (Trusty). You 
+can switch to Trusty from "Project Settings" -> "Build Environment" of your 
+project.
```
Comment (felicianotech) [inline jekyll/_docs/build-image-trusty.md]: Some people may not longer be on Ubuntu 14.04 and want to switch back.

Some people who had an existing project before the switch was made, are still using Ubuntu 12.04 because that was the default. Now they may want to switch to the new default image.

The goal of this PR is to simply remove the reference of Ubuntu 12.04 being the default since that is now false.

Comment (smart-alek) [inline jekyll/_docs/build-image-trusty.md]: Ok, if that's the case, then I think the best way to write this is:

"The default image is Ubuntu 14.04 (Trusty), but you can always change the image directly by going to 'Project Settings' -> 'Build Environment'."

This captures the (new) default and also shows people where to change that image if they need to.

File: jekyll/_docs/build-image-trusty.md (hunk 2 above)
Comment (smart-alek) [inline jekyll/_docs/build-image-trusty.md]: Right, but isn't that for code, not prose?

File: jekyll/_docs/build-image-trusty.md (hunk 4 above)
Comment (felicianotech) [inline jekyll/_docs/build-image-trusty.md]: Fixed in newest commit.

File: jekyll/_docs/build-image-trusty.md [hunk 5]
```diff
@@ -3,11 +3,13 @@ layout: classic-docs
...
 categories: [build-images]
 description: Ubuntu 14.04 (Trusty)
+changefreq: "weekly"
 ---
 
-You can run your Linux builds on Ubuntu 14.04 Trusty (default is Ubuntu 12.04). You can switch to Trusty from "Project Settings" -> "Build Environment" of your project.
+The default image is Ubuntu 14.04 (Trusty), but you can always change the image directly by going to 'Project Settings' -> 'Build Environment'.
 
-Please note that you need to trigger a build by pushing commits to GitHub or Bitbucket (instead of rebuilding) to apply the new setting.
+Please note that you need to trigger a build by pushing commits to GitHub or
```
Comment (smart-alek) [inline jekyll/_docs/build-image-trusty.md]: "To apply the new setting, you will need to trigger a build by pushing commits to GitHub or Bitbucket (instead of rebuilding)."

Comment (felicianotech) [inline jekyll/_docs/build-image-trusty.md]: We should just do this in another PR instead of continuing to delay this one. That's outside the point of this PR.

This point of which image is default is incorrect right now.
